checker/check-api-deprecated.go

```diff
@@ -0,0 +1,42 @@
+package checker
+
+import (
+	"github.com/tufin/oasdiff/diff"
+)
+
+func APIDeprecatedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config BackwardCompatibilityCheckConfig) []BackwardCompatibilityError {
```

Comment:
In some cases this change will be considered BREAKING (see check-api-deprecation.go) and it appear both as an ERROR and INFO.
I think we should only display INFO if it isn't breaking.

Reply:
In this [commit](https://github.com/Tufin/oasdiff/pull/260/commits/64ec3b278584dc70d961f2c22e1ef05cbe453705) I moved the INFO checks from APIDeprecatedCheck into APIDeprecationCheck to avoid confusion (both handling cases when an API is deprecated) and ending up with the same change presented in 2 ways (as deprecated with level `info` and then `breaking` if for example the sunset is not correctly configured).

Please let me know if you agree with this change, otherwise I can revert it and implement a mechanism based on error codes exclusions (e.g. code X will exclude code Y if both are detected for the same endpoint)